not-supported/borgbackup.sh

```diff
@@ -445,7 +445,7 @@ Please don't restart or shutdown your server until then!"
 
     # Prune system archives
     inform_user "$ICyan" "Pruning the system archives..."
-    if ! borg prune --prefix '*_*-NcVM-system-partition' "${BORG_PRUNE_OPTS[@]}"
+    if ! borg prune --glob-archives '*_*-NcVM-system-partition*' "${BORG_PRUNE_OPTS[@]}"
```

Comment:
The asterisk at the end is not needed IIRC
```suggestion
    if ! borg prune --glob-archives '*_*-NcVM-system-partition' "${BORG_PRUNE_OPTS[@]}"
```

Reply:
I'm not sure about that. The warning says that `--glob-archives 'yourprefix*'` should be used instead, so I assumed that it is necessary.

Reply:
Actually the string is a suffix so it should be fine to remoce the asterisk at the end